opentnsim/core.py

```diff
@@ -1049,13 +706,13 @@ def __init__(self, v=4, *args, **kwargs):
         self.v = v
         self.wgs84 = pyproj.Geod(ellps="WGS84")
 
-    def move(self):
+    def move(self, simulation_start):
```

Comment:
I don't expect this parameter here.  simulation start should be in the environment.